platform-db-runtime/src/main/java/com/softicar/platform/db/runtime/table/configuration/DbTableConfigurationCodeValidator.java

```diff
@@ -57,35 +60,53 @@ private DbTestDatabase createTestDatabase() {
 		return testDatabase;
 	}
 
-	private boolean isEnabled(JavaClasspath classPath, Optional<String> integrationProjectName, Optional<String> tablePackagePrefix) {
+	private Collection<IDbTable<?, ?>> findAllTables() {
+
+		var tables = new ArrayList<IDbTable<?, ?>>();
+		for (var root: JavaClasspath.getInstance().getPayloadRoots()) {
+			for (var javaClass: root.getAnalyzedClasses()) {
+				findTable(javaClass).ifPresent(tables::add);
+			}
+		}
+		return tables;
+	}
+
+	private Optional<IDbTable<?, ?>> findTable(AnalyzedJavaClass javaClass) {
```

Comment:
this method is partly redundant to `DbTableFinder`, e.g. it searches for `TABLE_FIELD_NAME`, etc. ... can we somehow unify this?

Reply:
did so, saved some lines of code